src/docs/patterns/empty-states.mdx

```diff
@@ -1,64 +1,89 @@
+The text on the screen should clearly indicate why there is no data to display, and what (if anything) can be done to make data available. The text should be minimal, but enough to convey the necessary message.
 
-### Refreshing Data
+If the data is empty but could be populated by a user action (such as adding a device to an empty device list), there should be a call-to-action button that will allow the user to add data.
 
-When refreshing data, there are several approaches you can take. If you are showing real-time data (either via polling or a push mechanism from the server), there is no need to show large loading indicators. You may wish to show a small indicator or at least mention when the data was last refreshed so users can identify if the data is stale.
+<ImageGrid images={[Image1, Image2]} />
 
-If your users can manually refresh, you may wish to show the placeholder elements (as with initial load), or simply refresh the data behind the scenes (as with real-time). Ultimately, this should depend on how long it takes to refresh the data.
+## Variations
 
-## Empty States
+### Full-screen
 
-<ImageGrid images={[Image3, Image4]} />
-<MaterialDesignDescription title={'Empty States'} url={'https://material.io/design/communication/empty-states.html'} />
+In the most common cases, Empty States should be used full-screen as the only content on a page. In this situation, the indicator should be centered on the page. In some cases, you may wish to use a large graphic as the background of the entire page as well (such as for features that will be coming soon).
 
-<!-- @TODO switch empty-state links over to the "Component Recommendations style" instead of a "Interactive Code example" once we have one designed. -->
+### Element-level
 
-<DemoCard
-    repository={'empty-states'}
-    angular={'https://pxblue-components.github.io/angular/?path=/story/components-empty-state--with-full-config'}
-    react={'https://pxblue-components.github.io/react/?path=/story/components-empty-state--with-full-config'}
-    ionic
-    reactNative
-/>
+While less common, empty states can also be used as an indicator for individual sections of a page rather than the entire page (such as a single card having no data on a dashboard / overview screen). In these cases, it may be appropriate to use a smaller icon and text depending on the available space.
 
-Empty states can be used in a variety of ways in your application. They can alert users to empty data or search results, lack of permissions, future features, or errors.
+<Divider />
 
-You can refer to the Material Design Empty States guidelines for general usage instructions.
+# Design Specifications
 
-Empty states should include a large icon or graphic, centered on the screen, followed a brief message. In some cases, you may wish to use a large graphic as the background of the entire page as well (such as for features that will be coming soon).
+<ImageGrid images={[Specs1Image, Specs2Image]} regularWidth />
```

Comment:
Not bad as far as the specs image goes ;) would be better if you can increase the size of that "100" red box

And I have always wondered why it is 100x100 instead of 96x96. Should we log a story to patch this?

Reply:
It is strange to use `<Device />` icon when the content says "no folders found". I have updated the empty states components in DSM (not published yet) using an outlined question mark icon. Use that instead.

Reply:
Yeah we can update the components to 96